packages/functional-tests/package.json

```diff
@@ -13,7 +13,7 @@
     "lint": "eslint ."
   },
   "devDependencies": {
-    "@playwright/test": "^1.35.1",
+    "@playwright/test": "^1.38.1",
```

Comment:
Remove `^` since I think that's bumping it up to 1.43.x, per https://semver.npmjs.com/